### PR TITLE
Run selenium suites in parallel

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Allow the selenium suites to run in parallel so we are able to run all of the tests with no added build time.
Also removing travis workaround as fix is now in.
https://docs.travis-ci.com/user/build-environment-updates/2016-12-06/#2016-12-06

### CODE REVIEW

File: tests/functional/config.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ let config = {
   services: ['selenium-standalone'],
   // Level of logging verbosity: silent | verbose | command | data | result | error
   logLevel: 'error',
+  bail: 1,
```
Comment (benjefferies) [inline tests/functional/config.js]: Not sure if we should do this, I find it quite useful to know the full state of the build on travis

File: tests/functional/config.js [hunk 2]
```diff
@@ -46,7 +47,7 @@ if (process.env.TRAVIS === 'true') {
   config = {
     ...config,
     logLevel: 'silent',
-    capabilities: [chrome]
+    capabilities: [firefox]
```
Comment (benjefferies) [inline tests/functional/config.js]: Why firefox?

Comment (jonnyshaw89) [reply tests/functional/config.js]: note the label "not-ready-to-merge"
don't waste your time looking at this yet.

File: tests/functional/config.js (hunk 1 above)
Comment (jonnyshaw89) [reply tests/functional/config.js]: doesn't work anyway ;)

**Comment by collisdigital** [comment]
Anyway to split tests into 4 (5 mins each?)

**Comment by jonnyshaw89** [comment]
@collisdigital the more we split this out the more we will hit the concurrent build limit.

**Review by benjefferies — APPROVED** [review]

**Review by iwootten — APPROVED** [review]
👍

**Pull request merged**